server/views/pages/overview/partials/overviewTab/_educationAndTrainingSummaryCard.njk

```diff
@@ -0,0 +1,92 @@
+{# Nunjucks template to display a summary card of the prisoner's eduction and training
```

Comment:
I've split the main `overviewTabContents` nunjucks templates into several components now, one for each of the main sections on the page (ie. goals summary card, education & training summary card, and session history summary card)
Each of these sections has it's own template file, with it's own unit test covering all paths 👍 
And each template describes the data that needs to be supplied to it via the view 👍